biofefi/components/configuration.py

```diff
@@ -263,3 +263,180 @@ def fi_options_box():
                     value=10,
                     key=ConfigStateKeys.NumberOfTopRules,
                 )
+
+
+@st.experimental_fragment
+def fi_options_page():
```

Comment:
:pick: Can you move this form to `biofefi.components.forms` and call it `fi_options_form` please?